src/renderer/lib/hooks/useHandleDragStart.ts

```diff
@@ -0,0 +1,23 @@
+import { ipcRenderer } from "electron";
+import { useToasts } from "react-toast-notifications";
+
+export const useHandleDragStart = () => {
+  const { addToast } = useToasts();
+
+  const handleDragStart = async (e: React.DragEvent<HTMLDivElement>, filePaths: string[]) => {
```

Comment:
It's best not to use single letter variable names for parameters.

```suggestion
  const handleDragStart = async (event: React.DragEvent<HTMLDivElement>, filePaths: string[]) => {
```